expirationd.lua

```diff
@@ -21,6 +22,7 @@ local function get_fiber_id(fiber)
 end
 
 local task_list = {}
+local cfg = {metrics = false}
```

Comment:
It is not enabled by default, when `metrics` module is present, as in `crud`?

Reply:
I misguide you, sorry. It is a bit different for `crud`: a user should enable statistics explicitly (`crud.cfg({stats = true})`), but `'local'` or `'metrics'` driver is choosen automatically depending on whether the statistics is enabled.

If we'll look on expirationd in those terms, our `'local'` driver is always enabled (it likely has low overhead). So we either enable `'metrics'` driver or not. Hm.

(Should we make `<...>.cfg()` options similar across modules? In theory, the answer should be 'yes', but there are differences like this 'always enabled' local driver. Interesting.)

Back to the question. There are cons and pros. If a statistics is just available on the default metrics dashboard in any project without explicit enabling it module-by-module, it is convenient. OTOH, we can add only low overhead statistics this way.

So I would ask it as: whether the overhead is low enough to enable it by default? I guess the answer is 'yes'.

However I has no enough level of confidence to insist on it. We need Georgy's vote for quorum :)

Reply:
@DifferentialOrange could you help us please?

Reply:
Yeah, the overhead is definitely low since the only operation that is performed by metrics is a callback call (it's several mathematical and assignment actions once in a minute or something, depends on monitoring agents). So no one should be affected by this in terms of performance

Reply:
Thank you, I've enabled it by default.